File: despatch.Domain/Models/Default/Response/ContactDataImageResponse.cs
namespace despatch.Domain.Models.Default.Response
{
    public class ContactDataImageResponse : BaseEntityResponse
    {
        public string ContactDataId { get; set; }
        public string ImageId { get; set; }

        public virtual ContactDataResponse ContactData { get; set; }
        public virtual ImageResponse Image { get; set; }
    }
}

File: despatch.WEB/app/src/swagger/models/contact-data-image-response.ts
/* tslint:disable */
import { BaseEntityResponse } from './base-entity-response';
import { ContactDataResponse } from './contact-data-response';
import { ImageResponse } from './image-response';
export interface ContactDataImageResponse extends BaseEntityResponse{
  contactData?: ContactDataResponse;
  contactDataId?: string;
  image?: ImageResponse;
  imageId?: string;
}
